Mathlib/Combinatorics/SimpleGraph/Walk.lean

```diff
@@ -1123,12 +1123,7 @@ theorem exists_boundary_dart {u v : V} (p : G.Walk u v) (S : Set V) (uS : u ∈
 @[simp] lemma getVert_copy  {u v w x : V} (p : G.Walk u v) (i : ℕ) (h : u = w) (h' : v = x) :
     (p.copy h h').getVert i = p.getVert i := by
   subst_vars
-  match p, i with
-  | .nil, _ =>
-    rw [getVert_of_length_le _ (by simp only [length_nil, Nat.zero_le] : nil.length ≤ _)]
-    rw [getVert_of_length_le _ (by simp only [length_copy, length_nil, Nat.zero_le])]
-  | .cons hadj q, 0 => simp only [copy_rfl_rfl, getVert_zero]
-  | .cons hadj q, (n + 1) => simp only [copy_cons, getVert_cons_succ]; rfl
+  rfl
```

Comment:
This could be fine; somebody more familiar with this file should check.

Reply:
Not familiar with the file either but this does look fine to me. This seems to be a standard "casting behaves as expected" lemma, and those are usually just `subst_vars; rfl`.

Reply:
perhaps @kmill can confirm that this change is fine?